handler/pom.xml

```diff
@@ -55,6 +55,11 @@
       <artifactId>netty-transport</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>netty-transport-native-unix-common</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
Instead of adding this dependency, does it make sense to move the `UnixChannel` to the `transport` module?
What do you think, @normanmaurer?

Reply:
Not sure... imho its not a big deal to just add the dependency

Reply:
@chrisvest so I think this is fine ?